<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <widget class="QPushButton" name="initiativeButton">
      <property name="text">
       <string>Roll for initiative!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="selectedLabel">
      <property name="text">
       <string>Selected</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4" rowspan="3">
     <widget class="QListWidget" name="initiativeListWidget"/>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="initiativeLabel">
      <property name="text">
       <string>Initiative</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="monsterLabel">
      <property name="text">
       <string>Monsters</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="playerLabel">
      <property name="text">
       <string>Players</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="addPlayerButton">
      <property name="text">
       <string>Add New Player</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QListWidget" name="playerListWidget"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QListWidget" name="monsterListWidget"/>
    </item>
    <item row="3" column="2" rowspan="3">
     <widget class="QListWidget" name="selectedListWidget"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="addMonsterButton">
      <property name="text">
       <string>Add New Monster</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
